processor/resourcedetectionprocessor/README.md

```diff
@@ -508,11 +508,33 @@ processors:
     override: false
 ```
 
+### Kubeadm Metadata
+
+Queries the K8S api server to retrieve the following node resource attributes:
+
+    * k8s.cluster.name
+
+The following permissions are required:
+```yaml
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list"]
+```
+
+| Name | Type | Required | Default         | Docs                                                                                                                                                                                                                                   |
+| ---- | ---- |----------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| auth_type | string | No       | `serviceAccount` | How to authenticate to the K8s API server.  This can be one of `none` (for no auth), `serviceAccount` (to use the standard service account token provided to the agent pod), or `kubeConfig` to use credentials from `~/.kube/config`. |
+
 ### K8S Node Metadata
 
 Queries the K8S api server to retrieve the following node resource attributes:
 
     * k8s.node.uid
+    * k8s.node.name
```

Comment:
looking at the code of the k8s node detector, it supports fetching the k8s node uid and name, since name was not present in the documentation (probably just due to forgetting to add it there) I added it. Since it's such a small fix in docs, I think it can be added as part of this PR

Reply:
That's fair, just wanted to make sure we weren't missing anything.